### ISSUE TITLE
Distinction between categorical_crossentropy and sparse_categorical_crossentropy is not clear in the documentation

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorflow#49379]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/keras/losses/categorical_crossentropy
https://www.tensorflow.org/api_docs/python/tf/keras/losses/sparse_categorical_crossentropy

## Description of issue (what needs changing):

There is documentation for these 2 losses but there is no clear explanation distinguishing each of them and when exactly a specific Loss should be used.

**Comment by ymodak** [comment]
A closer look at the usage example and their args documentation should help to spot the difference between the two functions.
[`y_true` arg takes `one hot encoded` values vs `ground truth` values for sparse]
Thanks!

**Comment by worldpeaceaspirer** [comment]
@ymodak,
Instead of taking a closer look at the code, a line of description highlighting the difference would make it more obvious and clear.

**Comment by ymodak** [comment]
IMHO I disagree, reason being that there are widespread areas like these where you can argue that functions description is limited but they can be easily understood digging into usage examples. 
Thanks!

**Comment by worldpeaceaspirer** [comment]
@ymodak,
It's very harsh to close an issue without a proper justification. Just that you disagree, it doesn't mean it should be closed abruptly. 
I am not happy or not satisfied.

**Comment by ymodak** [comment]
I think my justification is clear, the docs for those functions are good enough for any user to understand thier usages.
You don't need a distinction between different functions to justify further since the topics are independently and coherently discussed on their corresponding documentation page.
If your query was of a kind where api docs are limited that would have been a different case to deal with, here you are randomly comparing two functions and adding distinction between them which is not the design for any other docs anywhere on the website.
Therefore the issue was closed. Hope this makes sense. Thanks!
